src/test/resources/frequencyColumnCompressionData.txt

```diff
@@ -0,0 +1,80 @@
+table datawave.metadata
```

Comment:
I think we can delete this file.  Not used and incorrect values (should not be strings).

Reply:
Sure.  Delete away.  I thought it could be useful at one point in time.  Probably used it in early tests.